inputs.tf

```
@@ -45,24 +33,29 @@ variable "master_ipv4_cidr_block" {
   description = "The IP range to set for master nodes, passed to master_ipv4_cidr_block - /28 required by Google."
 }
 
-variable "min_master_version" {
+variable "maximum_node_count" {
   type        = string
-  description = "The min_master_version attribute to pass to the google_container_cluster resource."
+  description = "Maximum nodes for the node pool. This is the total nodes so for regional deployments it is the total nodes across all zones."
 }
 
-variable "minimum_node_count" {
+variable "machine_type" {
   type        = string
-  description = "Minimum nodes for the node pool. This is the total nodes so for regional deployments it is the total nodes across all zones."
+  description = "Machine types to use for the node pool."
```

```suggestion
  description = "Machine type to use for the node pool."
```

Only supports a single type I presume.